docs/lib/analytics/fragments/ios/identifyuser.md

```diff
@@ -1,5 +1,4 @@
-This call sends information that you have specified about the user to Amazon Pinpoint. This could be an authenticated or guest user. AWSMobileClient assigns all users an `identityId` that can be used to call `Amplify.Analytics.identifyUser`. If you have asked for location access and received the user's location information, you can also provide that in `AnalyticsUserProfile.Location`.
-
+This call sends information that you have specified about the user to Amazon Pinpoint. This could be an authenticated or guest user. AWSMobileClient assigns all users an `identityId` that can be used to call `Amplify.Analytics.identifyUser()`. If you have asked for location access and received the user's location information, you can also provide that in `AnalyticsUserProfile.Location`.
```

Comment:
suggestion: `AWS Pinpoint` instead of `Amazon Pinpoint`

Reply:
Good call, will change

Reply:
Why is this change being suggested? The service's name is *Amazon Pinpoint*.

Reply:
Good call, I didn't realize it since in our SDKs we call it `AWSPinpoint`. I'll change back to Amazon Pinpoint and make sure it's consistent in all docs.